### DISCUSSION TITLE
5.1音轨 用2.1音响播放    环境声比对白声音大很多    有没有什么好的解决办法

### DISCUSSION THREAD

**Discussion by 906018080** [hooke007/mpv_PlayKit#182] [mpv 通用]
ad-lavc-downmix=yes				#声道下混处理(默认:no)
audio-channels=stereo				#下混为立体声
audio-normalize-downmix=yes		#立体声规格化(默认no)
试了试这几个设置,感觉没有什么明显变化,总音量反而越来越小

或者说有没有相关命令或者脚本可以自定义声道混合的 设置哪一通道进左或者右 然后混进去多少量?

**Comment by natural-harmonia-gropius** [comment]
用 windows 自带的空间音频或者其他通过 hrtf 实现的方式来做混合，简单的 downmix 不如听 2.0 音轨
![image](https://user-images.githubusercontent.com/50797982/190897827-32e78e3f-a5cb-45de-adfd-55ab3275db78.png)

**Reply by 906018080** [reply]
现在影视资源大都是5.1甚至7.1,主要是没得选
好在我在你 实用向滤镜 文档下面 翻到了相关音频滤镜命令   有用但是快进时候有点卡  我先去研究研究相关参数
